sensorflux/database_connector.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+sensorflux.database_connector
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module contains the class that communicates data to influxdb.
+"""
+
+from copy import deepcopy
+from datetime import datetime
+
+from influxdb import InfluxDBClient
+
+
+class DatabaseConnector:
+    """
+    A class to represent a client connection to the influxdb instance.
+
+    :param str measurement: measurement name.
+    :param str device: name for the device tag.
+    :param tuple fields: names of the fields containing the data to be sent.
+    """
+    def __init__(self, measurement, device, fields):
+        self.client = InfluxDBClient(
```

Comment:
I'd move the constants you use here to class member variables for clarity and for the unlikely event that someone requires them to be changed

Reply:
The visibility of .client should be "protected", along the members that follow below

Reply:
> I'd move the constants you use here to class member variables for clarity and for the unlikely event that someone requires them to be changed

done, thanks for the review! i was also thinking about issue #3 there, getting these constants directly from a config once there is one